moveit_simple_msgs/srv/TrajectoryTrainer.srv

```diff
@@ -0,0 +1,19 @@
+uint8 NEW_WP=0				# Create a new waypoint at current location 			[required: waypoint name, optional: trajectory name]
+uint8 DEL_WP=1				# Delete existing waypoint(s) from config 				[required: waypoint name(s)]
+uint8 GET_WP=2				# Request information on existing waypoint(s) 			[required: waypoint name(s)]
+uint8 UPDATE_WP=3			# Replace previous waypoint data with current location 	[required: waypoint name]	
+uint8 APPEND_WP_TO_TRAJ=4	# Add waypoint(s) to trajectory                         [required: trajectory name, waypoint name(s)] 
+uint8 REMOVE_WP_FROM_TRAJ=5 # Remove waypoint(s) from trajectory if it(they) exist 	[required: trajectory name, waypoint name(s)]
+uint8 NEW_TRAJ=10			# Create a new trajectory, optionally populate          [required: trajectory name, optional: waypoint name(s)]
+uint8 DEL_TRAJ=11			# Delete a trajectory from config						[required: trajectory name]
+uint8 GET_TRAJ=12			# Get all waypoints in a trajectory 					[required: trajectory name]
+uint8 CLEAR_TRAJ=13			# Clears the contents of a trajectory 					[required: trajectory name]
+uint8 RELOAD_YAML=255		# Update internal storage to match YAML                 [required: N/A]
+
+uint8 op_type
+string trajectory_name
+string[] waypoint_names
+---
+bool result
+string message
```

Comment:
What is the `message` member for?

Reply:
Message is intended to provide diagnostic information when an operation is unsuccessful. Generic name allows it to be used for user-specific purposes without too much confusion.